### PR TITLE
Wlssb: Even More Updates

### PR DESCRIPTION
**Pull request by Lycanium-Z**
Thanks @ragiingalways for finally explainging what that annoying crash was about

### CODE REVIEW

**Comment by Lycanium-Z** [comment]
Tested and works

File: mods/wlssb/abilities.js [hunk 1]
```diff
@@ -322,9 +325,9 @@ exports.BattleAbilities = {
 	muscles: {
 		id: "muscles",
 		name: "Muscles",
-		desc: "+2 defense, +2 Special defense, -3 attack, +1.5 special attack on switch in.",
+		desc: "+3 defense, +3 Special defense, -3 attack, +1.5 special attack on switch in.",
 		onStart: function (pokemon) {
-			this.boost({atk: -4, def: 2, spa: 1, spd: 2});
+			this.boost({atk: -4, def: 4, spa: 1, spd: 4});
```
Comment (RagiingAlways) [inline mods/wlssb/abilities.js]: Doesn't match Desc.

**Comment by Lycanium-Z** [comment]
Ok so need a new staffmon then .-.

**Comment by Lycanium-Z** [comment]
Added SnorlaXTherain (thx KM <3). removed alll traces of evasion boosts. redid mine

File: mods/wlssb/abilities.js [hunk 2]
```diff
@@ -277,7 +277,7 @@ exports.BattleAbilities = {
 	felinefury: {
 		id: "felinefury",
 		name: "Feline Fury",
-		desc: "+3 Attack on switch in.",
+		desc: "+2 Attack on switch in + Scrappy",
```
Comment (DeathlyPlays) [inline mods/wlssb/abilities.js]: Did you update the entry?

**Comment by HoeenCoder** [comment]
Use `npm test` to test your commits (its the same thing travis runs)

**Comment by Lycanium-Z** [comment]
I did. Travis buddy is being weird.

File: mods/wlssb/abilities.js (hunk 2 above)
Comment (Lycanium-Z) [reply mods/wlssb/abilities.js]: yes yes i did

**Comment by Lycanium-Z** [comment]
er anyways everything is tested and works @hoeencoder

**Comment by Lycanium-Z** [comment]
bop @HoeenCoder

**Pull request merged**